test/vni-manager-mocha.js

```diff
@@ -194,6 +195,135 @@ describe("vni-manager", function() {
             expect(testVni.outputState.previousLms).to.be.undefined;
         });
 
+        describe("#clearTransientInputs", function() {
+
+            it("should clear packet input on a VNI", function(done) {
+
+                var présenter = "Puis-je me présenter";
+
+                return test.createNetwork(
+
+                    { inputNode: 'core/Repeat',
+                      leNode: 
+                         pipelineFactory({ 
+                             description: "Un p'tit node avec un edge",
+                             inPorts: {
+                                 input: {
+                                     datatype: 'string',
+                                     required: true
+                                 }
+                             }
+                          },
+                          {fRunUpdater: function(vni) {
+                               // verify we have the correct state on the VNI input port
+                               test.verifyState(vni.inputStates()['input'], '', présenter);
+
+                               // Clear the non-IIP port state(s), i.e., the input port state
+                               vni.clearTransientInputs();
+                               expect(vni.inputStates()['input']).to.be.undefined;
+
+                               done();
+                          }
+                         })
+
+                }).then(function(network) {
+
+                    var leNode = network.processes.leNode.component;
+                    network.graph.addEdge('inputNode', 'out', 'leNode', 'input');
+                    network.graph.addInitial(présenter, 'inputNode', 'in');
+
+                });
+            });
+
+            it("should not fail if socket is not initialized yet", function(done) {
+
+                var enchanté = "Enchanté de faire votre connaissance";
```

Comment:
OK.  I was curious how well it would handle the extended latin set, which is why I used it.   And it works quite well.  However, since it concerns you, I will change it.